packages/sdk-components-react-remix/src/form.ws.tsx

```diff
@@ -30,59 +32,103 @@ export const meta: WsComponentMeta = {
     {
       type: "instance",
       component: "Form",
-      children: [
-        {
-          type: "instance",
-          component: "Label",
-          children: [{ type: "text", value: "Name" }],
-        },
+      props: [
         {
-          type: "instance",
-          component: "Input",
-          props: [{ type: "string", name: "name", value: "name" }],
-          children: [],
-        },
-        {
-          type: "instance",
-          component: "Label",
-          children: [{ type: "text", value: "Email" }],
-        },
-        {
-          type: "instance",
-          component: "Input",
-          props: [{ type: "string", name: "name", value: "email" }],
-          children: [],
-        },
-        {
-          type: "instance",
-          component: "Button",
-          children: [{ type: "text", value: "Submit" }],
+          name: "state",
+          type: "string",
+          value: "initial",
+          dataSourceRef: {
+            type: "variable",
+            name: "formState",
+          },
         },
+      ],
+      children: [
         {
           type: "instance",
-          component: "SuccessMessage",
+          label: "FormContent",
+          component: "Box",
+          props: [
+            {
+              name: showAttribute,
+              type: "boolean",
+              value: false,
+              dataSourceRef: {
+                type: "expression",
+                name: "formInitial",
+                code: `formState === 'initial' || formState === 'error'`,
+              },
+            },
+          ],
           children: [
             {
               type: "instance",
-              component: "Text",
-              children: [
-                { type: "text", value: "Thank you for getting in touch!" },
-              ],
+              component: "Label",
+              children: [{ type: "text", value: "Name" }],
+            },
+            {
+              type: "instance",
+              component: "Input",
+              props: [{ type: "string", name: "name", value: "name" }],
+              children: [],
+            },
+            {
+              type: "instance",
+              component: "Label",
+              children: [{ type: "text", value: "Email" }],
+            },
+            {
+              type: "instance",
+              component: "Input",
+              props: [{ type: "string", name: "name", value: "email" }],
+              children: [],
+            },
+            {
+              type: "instance",
+              component: "Button",
+              children: [{ type: "text", value: "Submit" }],
             },
           ],
         },
+
         {
           type: "instance",
-          component: "ErrorMessage",
+          label: "SuccessMessage",
```

Comment:
any reason to use camel case instead  of 2 capitalized words?

Reply:
I guess no just mimicking previous names